modules/apps/portal-search/portal-search-admin-web/src/main/resources/META-INF/resources/css/_index_actions.scss

```diff
@@ -0,0 +1,9 @@
+.search-admin-actions-panel {
+	.float-left {
```

Comment:
@thektan quick question, im backporting this to 7.2.x, i should use .pull-left right?
https://github.com/BryanEngler/liferay-portal/blob/7.2.x/modules/apps/portal-search/portal-search-admin-web/src/main/resources/META-INF/resources/css/main.scss#L51

Reply:
Just checked and ya `.pull-left`. Nice catch. I just moved the css over. 👍